docs/CHANGELOG.md

```diff
@@ -130,7 +134,7 @@ title: Changelog
 
 * Ensure consistent indentation with Rubocop.
 
-    *Joel Hawksley
+    *Joel Hawksley*
```

Comment:
looks like this change is irrelevant to the PR?

Reply:
I'm fine with landing this formatting fix :)